src/Aspire.Hosting/Extensions/ContainerResourceBuilderExtensions.cs

```diff
@@ -39,7 +39,8 @@ public static IResourceBuilder<ContainerResource> AddContainer(this IDistributed
     }
 
     /// <summary>
-    /// Adds a binding to expose an endpoint on a resource.
+    /// Exposes an endpoint on a resource. This binding reference can be retrieved using <see cref="ResourceBuilderExtensions.GetEndpoint{T}(IResourceBuilder{T}, string)"/>.
+    /// The binding name will be the scheme name if not specified.
```

Comment:
I think we should swap out "binding" with "endpoint" in all of these, e.g.

```suggestion
    /// The endpoint name will be the scheme name if not specified.
```